src/LinuxTracing/PerfEvent.h

```diff
@@ -343,6 +343,7 @@ struct SchedWakeupWithStackPerfEventData {
   std::unique_ptr<uint64_t[]> regs;
   uint64_t dyn_size;
   std::unique_ptr<uint8_t[]> data;
+  bool just_tracepoint;
```

Comment:
I don't understand this, what does it mean? That the callstack is not present maybe? But don't we already have the existing type for the case without callstack?